Sources/SwiftParserDiagnostics/ParseDiagnosticsGenerator.swift

```diff
@@ -459,6 +459,29 @@ public class ParseDiagnosticsGenerator: SyntaxAnyVisitor {
       )
       return .visitChildren
     }
+    if let unexpectedBetweenArgumentAndRightParen = node.unexpectedBetweenArgumentAndRightParen,
+      var missingArgumentNode = node.argument?.missingNodes.only
```

Comment:
`missingNodes` seems a little magical to me and I get the feeling that it’s not the right design because you need to patch it up further below. I think a better design would be to

- Modify the parser and eat e.g. `0xff` as an unexpected token when parsing the `VersionTupleSyntax`.
- Generate the diagnostic in `visit(_ node: VersionTupleSyntax)`

Reply:
Aha I'll try again. Thanks for the feedback! 🙇